### DISCUSSION TITLE
Is there a way to embed content in a panel in Swell's free version?

### DISCUSSION THREAD

**Discussion by maxklema** [swellstores/community#13] [Help] [answered]
Hey,

I am trying to embed an Instagram feed into a client's home page, but there doesn't seem to be any place to embed content. Is there a way to do this?

If not, is it possible for this to be added?

**Comment by greph** [comment, marked as answer]
Hey @agileinx , are you looking to embed the content within a Swell-hosted storefront, like Origin? Also, are you using the storefront editor?

**Reply by maxklema** [reply]
Yes, I am using the store-front editor and a swell-hosted storefront.

**Reply by greph** [reply]
Hey @agileinx , custom sections can be added to the editor by performing the following actions in your local repository and pushing it back up to your Swell hosted storefront:

1. Copy over one of the json files in config/content. Update the “name” and “slug” fields and update the included fields as needed
2. Add the new section to the “item_types” array found in page.json in config/content
3. Commit and push back your changes. The new section should now appear in the list of new Section types within the editor.

Alternatively, these fields can be updated programmatically within the /:content and /:content/theme.origin.page endpoints respectively.
